Sources/MPComponents/Natives/Button/MPButtonStyle.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  PrimaryButtonStyle.swift
+//  MercadoPagoSDK
+//
+//  Created by Guilherme Prata Costa on 23/06/25.
+//
+import SwiftUI
+import MPFoundation
+
+struct MPButtonStyle: ButtonStyle {
+    enum Variant {
+        case loud
+        case quiet
+        case transparent
+    }
+
+    enum Size {
+        case large
+        case medium
+    }
+    
+    @Environment(\.checkoutTheme) var theme: MPTheme
+    @Environment(\.isEnabled) private var isEnabled: Bool
+    
+    let variant: Variant
+    let size: Size
+    
+    public func makeBody(configuration: Configuration) -> some View {
+        let variantAppearance = getVariantAppearance()
+        let sizeMetrics = getSizeMetrics()
+        
+        let currentBackgroundColor = isEnabled ?
+        (configuration.isPressed ? variantAppearance.pressedBackgroundColor : variantAppearance.backgroundColor)
+        : variantAppearance.disabledBackgroundColor
+        
+        let currentForegroundColor = isEnabled ?
+        (configuration.isPressed ? variantAppearance.pressedForegroundColor : variantAppearance.foregroundColor)
+        : variantAppearance.disabledForegroundColor
+
+        HStack {
+            configuration.label
+        }
+        .padding(sizeMetrics.padding)
+        .font(sizeMetrics.font)
+        .foregroundColor(currentForegroundColor)
+        .background(currentBackgroundColor)
+        .overlay(
+            RoundedRectangle(cornerRadius: variantAppearance.cornerRadius)
+                .stroke(currentBackgroundColor, lineWidth: variantAppearance.borderWidth)
+        )
+        .cornerRadius(variantAppearance.cornerRadius)
+        .animation(.easeOut(duration: 0.15), value: configuration.isPressed)
+        .animation(.easeOut(duration: 0.15), value: isEnabled)
+    }
+    
+    private func getVariantAppearance() -> MPButtonAppearance {
+        switch variant {
+        case .loud: return theme.buttons.loud
+        case .quiet: return theme.buttons.quiet
+        case .transparent: return theme.buttons.transparent
+        }
+    }
+    
+    private func getSizeMetrics() -> MPButtonSize {
+        switch size {
+        case .large: return theme.buttons.sizes.large
+        case .medium: return theme.buttons.sizes.medium
+        }
+    }
+}
+
+
+struct ContentView: View {
+    @State private var size: MPButtonStyle.Size = .large
+    
+    var body: some View {
+        VStack(alignment: .center) {
+            Spacer()
+                        
+            Text("Button Style - Loud")
+            Group {
+                Button("Label") {
+                    print("Botão pressionado!")
```

Comment:
could we have the logs in english to maintain consistency?

Reply:
Sure, i wiil change